hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/rest/api/QuestionAnswersRelation.java

```diff
@@ -40,11 +41,12 @@
 @RelationshipResource(name = "answers", title = "Answers to questions about documents", entityResource = QuestionsEntityResource.class)
 public class QuestionAnswersRelation implements RelationshipResourceAction.Read<AnswerModel>
 {
+    private final HxInsightClient hxInsightClient;
 
     @WebApiDescription(title = "Get answers to a question")
     public CollectionWithPagingInfo<AnswerModel> readAll(String questionId, Parameters parameters)
     {
-        AnswerModel answerModel = new AnswerModel("Dummy answer");
-        return CollectionWithPagingInfo.asPagedCollection(answerModel);
+        hxInsightClient.getAnswer(questionId);
+        return CollectionWithPagingInfo.asPagedCollection(new AnswerModel("Dummy answer"));
```

Comment:
why don't you pass answer to client?

Reply:
I'll do that when I work on the next task in the story. Will also add some more tests (including integration test I hope).